packages/turbo/ios/RNSessionConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RNSessionConfiguration.swift
+//  RNTurbo
+//
+//  Created by Patryk Klatka on 22/04/2024.
+//
+
+import Foundation
+
+class RNSessionConfiguration {
```

Comment:
Something bugs me with the naming. Are we configuring a Session? I don't think so.